Bot.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import telebot
+import ORMForBot as ORM
+from telebot import types
+import TicTacToeForBot as TicTacToe
+bot = telebot.TeleBot('5243687518:AAEYQJGdmCx86OvyIRTzAjDXwN1lM0dgFXU');
+nick = ''
+winner = 2
+theBoard = [' ']*10
+playerLetter, computerLetter = ['X', 'O']
+@bot.message_handler(content_types=['text'])
+
+def start_bot(message):
```

Comment:
There should be 2 blank lines before and after function definition

Reply:
Done